tensorboardX/x2num.py

```diff
@@ -19,6 +16,16 @@ def makenp(x, modality=None):
         return mxnet_np(x, modality)
 
 
+def numpy_np(x, modality):
```

Comment:
In this context, xxx_np means convert datatype of `xxx` to `numpy`.

Reply:
Yeah. But without it `tensor` and `ndarray` behavior is confusing when one uses `SummaryWriter.add_image(...)`. The reason is that because `_prepare_image(x)` flips second and zero-th index.

For example these would behave identically:
```python
writer.add_image(np.random.rand([W, H, C])
writer.add_image(torch.rand([C, W, H])
```

So I tried to make them behave the same, but the function name came out a bit strange because I followed the same patter as the other functions.

Reply:
How about changing all xxx_np to prepare_xxx?

Reply:
I'm okay with that. I'm kind of a new kid on the block... So I don't really have a naming preference yet 😉 Other than that is there anything else blocking the merging for you?

Reply:
Looks good!